monitor/services/src/main/java/org/datacleaner/monitor/scheduling/SchedulingService.java

```diff
@@ -42,7 +42,8 @@
 public interface SchedulingService extends RemoteService {
 
     @RolesAllowed({ SecurityRoles.VIEWER, SecurityRoles.SCHEDULE_EDITOR })
-    public List<ScheduleDefinition> getSchedules(TenantIdentifier tenant) throws DCSecurityException;
+    public List<ScheduleDefinition> getSchedules(TenantIdentifier tenant, boolean loadProperties) 
+            throws DCSecurityException;
```

Comment:
Not really a concern. It's a GWT class, so it not a public API (yet, hopefully at the end of the monitor rewrite, all interfaces to the web app are public)